package.json

```diff
@@ -22,6 +21,7 @@
     "react-beautiful-dnd": "^13.1.1",
     "react-dom": "^18.2.0",
     "react-icons": "^4.10.1",
+    "react-pdf": "^7.3.3",
```

Comment:
POTENTIALLY BLOCKING: Are you actually using this? I don't see you import it anywhere? This is usually used to convert React code to a PDF rather than to view a PDF that is uploaded in PDF format.

Reply:
yeah I wanted to see it in VSCode but that still didn't work. I'll remove it